x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/use_explore_data.ts

```diff
@@ -80,8 +86,29 @@ export const useExploreData = (
 
       try {
         const resultsField = jobConfig.dest.results_field;
+        const searchQueryClone: ResultsSearchQuery = cloneDeep(searchQuery);
+        let query: ResultsSearchQuery;
+
+        if (JSON.stringify(searchQuery) === JSON.stringify(defaultSearchQuery)) {
+          query = {
+            exists: {
+              field: resultsField,
+            },
+          };
+        } else if (isResultsSearchBoolQuery(searchQueryClone)) {
+          searchQueryClone.bool.must.push({
```

Comment:
is `bool.must` guaranteed to exist in the cloned `searchQuery`?

Reply:
Good question - all the queries produced by the eui search bar so far do but there isn't any documentation I could find that guarantees it. Better to be safe. Added check in cf69901043df54a047c4c010c52970e7236e944c